script/main.py

```diff
@@ -243,17 +244,37 @@ def average_age(conn, gender):
     print('Average age of women: {value:.2f} years'.format(value = average_age_of_women))
   else:
     print('Overall average age: {value:.2f} years'.format(value = average_age_overall))
+
+
+def select_all_sorted_cities(conn):
+  cur = conn.cursor()
+  cur.execute("SELECT location_city, count(location_city) FROM users GROUP BY location_city ORDER BY count(location_city) DESC")
+  cities = cur.fetchall() 
+  return cities
+
+
+def most_popular_cities(conn, number):
+  cities = select_all_sorted_cities(conn)
+  i = 0
+  while i < int(number):
```

Comment:
Instead of this loop it is better to use
```
for i in range(number):
```
And number should already be passed as int to this function.

Notice that this loop will change anyway when you improve SQL command